sql/core/src/main/scala/org/apache/spark/sql/execution/limit.scala

```diff
@@ -26,7 +27,7 @@ import org.apache.spark.sql.catalyst.plans.physical._
 import org.apache.spark.sql.catalyst.util.truncatedString
 import org.apache.spark.sql.execution.exchange.ShuffleExchangeExec
 import org.apache.spark.sql.execution.metric.{SQLShuffleReadMetricsReporter, SQLShuffleWriteMetricsReporter}
-import org.apache.spark.util.ArrayImplicits._
+import org.apache.spark.sql.execution.python.HybridRowQueue
```

Comment:
On a side note, will we be moving xx to a more common package in the future?

Reply:
It was only used for python udf/udft/datasource/etc before, but I also feel it could be in `org.apache.spark.sql.execution`.
However, I don't feel very strong about it.